File: uri/1 - Basics/1013/Main.rb
def main()

    aux = 0
    ar=gets.split()
    a=ar[0].to_i
    b=ar[1].to_i
    c=ar[2].to_i

    if(a > b)
    	aux = a
    else
	    aux = b
    end
    
    if(aux > c)
        aux = aux
    else
        aux = c
    end
    
    print("%d eh o maior\n" % (aux) )

end

main()

File: uri/1 - Basics/1013/Main.cs
using System;

class URI
{

	static void Main(string[] args)
	{

		string[] vet = Console.ReadLine().Split(' ');
		int a = int.Parse(vet[0]);
		int b = int.Parse(vet[1]);
		int c = int.Parse(vet[2]);

        int aux=0;

		if(a > b)
            aux = a;
        else
            aux = b;
        
        if(aux < c)
            aux = c;
        
        Console.WriteLine($"{aux} eh o maior");
	}

}
